multirange.js

```diff
@@ -15,6 +15,8 @@ var multirange = function(input) {
 	var min = +(input.min || 0);
 	var max = +(input.max || 100);
 	var ghost = input.cloneNode();
+	var drag = input.getAttribute("data-drag-middle")!==null;
```

Comment:
You might want to change this variable name too, right now it's overly generic.
Also space around operators :)